src/components/profiles/DynamicProfileImg.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { GatsbyImage } from "gatsby-plugin-image"
+import * as style from "./ProfileImg.module.scss"
+
+interface Props {
+  data: DynamicImageData,
+  alt?: string
+}
+
+const DynamicProfileImg: React.FC<Props> = ({ data, alt }) => {
```

Comment:
Should we merge the ProfileImg components into one and handle the dynamic or static images based on the props given? I.e. if the img is a dynamic img object use dynamic image else use static image